### PR TITLE
Fastroping - Remove FRIES on vehicle deletion / adjust Ghosthawk Fries

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- Move FRIES removal logic to separate function
- Remove FRIES when vehicles are deleted
- Adjust FRIES bar position on Ghosthawk to not clip with right door (https://imgur.com/a/s4zAcZ2)
- Mark `equipFRIES` as public function, TODO documentation? Where?

### CODE REVIEW

File: addons/fastroping/XEH_PREP.hpp [hunk 1]
```diff
@@ -6,6 +6,7 @@ PREP(canPrepareFRIES);
 PREP(canStowFRIES);
 PREP(checkVehicleThread);
 PREP(cutRopes);
+PREP(deequipFRIES);
```
Comment (bux) [inline addons/fastroping/XEH_PREP.hpp]: unequipFRIES

**Comment by jonpas** [comment]
Documentation on Fastroping Framework wiki page.

**Comment by veteran29** [comment]
I assume this can be safely deleted as I've added `deleted` event handler:
https://github.com/acemod/ACE3/blob/master/addons/fastroping/functions/fnc_checkVehicleThread.sqf
?

**Comment by dedmen** [comment]
If we can trust the Deleted handler.. Yes.
In the past we only had problems with the deleted EH firing too often, while testing I didn't find a single occurence of it not firing. So it should be fine

**Comment by veteran29** [comment]
I think I will add `true/false` return to `equipFRIES `, if it is public it should give feedback if fries were added or not. Opinions?

**Comment by veteran29** [comment]
Added docs.

**Pull request merged**